src/demo_blank/configs/dipdup.swarm.yaml

```diff
@@ -11,7 +11,7 @@ hasura:
   url: http://${HASURA_HOST:-demo_blank_hasura}:8080
   admin_secret: ${HASURA_SECRET:-changeme}
   allow_aggregations: ${HASURA_ALLOW_AGGREGATIONS:-false}
-  select_limit: ${HASURA_SELECT_LIMIT:-100}
+  select_limit: ${HASURA_SELECT_LIMIT:-10000}
   camel_case: ${HASURA_CAMEL_CASE:-true}
```

Comment:
Maybe also change to camel_case: false? We always used snake_case for our indexers as far as I remember